### PR TITLE
Gevos/behig improvements

### PR DESCRIPTION
**Pull request by gevos**

### CODE REVIEW

File: src/components/Link/Link.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const defaultProps = {};
 
 const Link = ({ href, children }) => (
   <a className="wkp-link" href={href} rel="noopener noreferrer" target="_blank">
-    {children}
+    <span>{children}</span>&nbsp;
```
Comment (gevos) [inline src/components/Link/Link.js]: Core change together with style changes (is relevant for link in buslines layer info (lang=IT)

File: src/components/TopicMenu/TopicMenu.scss [hunk 2]
```diff
@@ -83,7 +83,11 @@
       color: #888;
       height: 30px;
       padding: 0;
-      margin: 0 4px 0 4px;
+      margin: 0 4px 1px 4px;
+
+      .tm-layer-tree-toggle {
+        line-height: 14px;
+      }
```
Comment (gevos) [inline src/components/TopicMenu/TopicMenu.scss]: Core changes to better style multiple multiline layer names

File: src/popups/BehigPopup/BehigPopup.js [hunk 3]
```diff
@@ -86,7 +80,7 @@ const BehigPopup = ({ feature, language, t }) => {
       })
       // Format sentences
       // eslint-disable-next-line react/no-array-index-key
-      .map((sentence, idx) => <div key={idx}>{sentence}.</div>);
+      .map((sentence, idx) => <div key={idx}>{sentence}</div>);
```
Comment (gevos) [inline src/popups/BehigPopup/BehigPopup.js]: I have removed the dot at the end as it would look strange with the arrow.

**Comment by edellucien** [comment]
Depends from react-spatial update: https://github.com/geops/react-spatial/pull/423/commits
Plus need to merge first: https://github.com/geops/trafimage-maps/pull/371
Because new style from react-spatial already fixed there.

File: src/components/TopicLoader/TopicLoader.js [hunk 4]
```diff
@@ -138,6 +138,29 @@ class TopicLoader extends Component {
...
+        if (lay.getIsBaseLayer()) {
+          if (idx === 0) {
+            lay.setVisible(true);
+          } else {
+            lay.setVisible(false);
+          }
+          return lay;
+        }
+        return lay;
+      });
```
[the remark this answers is not in the document]
Comment (edellucien) [reply src/components/TopicLoader/TopicLoader.js]: I added your suggestions to the commit 6985211ecc4f429af1ff3a8fd269524face06e55

**Pull request merged**